XLS-47d-PriceOracles/README.md

```diff
@@ -101,81 +102,58 @@ We define a new transaction **OracleSet** for creating or updating a `PriceOracl
 | `Account` | :heavy_check_mark: | `string` | `ACCOUNTID` |
-
-`Account` is the XRPL account that has update and delete privileges on the Oracle being set.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
-| `OracleSequence` | :heavy_check_mark: | `string` | `UINT32` |
-
-`OracleSequence` is a unique identifier of the Price Oracle for the given Account.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
+| `OracleDocumentID` | :heavy_check_mark: | `string` | `UINT32` |
 | `Provider` | | `string` | `BLOB` |
-
-`Provider` identifies an Oracle Provider. `Provider` must be included when creating a new instance of `PriceOracle`.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
 | `URI` | | `string` | `BLOB` |
-
-`URI` is an optional field to reference the price data off-chain.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
 | `AssetClass` | | `string` | `BLOB` |
-
-`AssetClass` describes the asset's type.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:-----------|
 | `LastUpdateTime` | :heavy_check_mark: | `number` | `UINT32` |
-
-`LastUpdateTime` is the specific point in time when the data was last updated. `LastUpdateTime` is represented in Unix Time.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
 | `PriceDataSeries` | :heavy_check_mark: | `array` | `ARRAY` |
-
-`PriceDataSeries` is an array of up to ten `PriceData` objects, where `PriceData` represents the price information for a token pair. `PriceData` includes the following fields:
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
 | `BaseAsset` | :heavy_check_mark: | `string` | `CURRENCY` |
-
-`BaseAsset` is the asset to be priced.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
 | `QuoteAsset` | :heavy_check_mark: | `string` | `CURRENCY` |
-
-`QuoteAsset` is the denomination in which the prices are expressed.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
 | `AssetPrice` | :heavy_check_mark: | `number` | `UINT64` |
-
-`AssetPrice` is the scaled asset price, which is the price value after applying the scaling factor.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
 | `Scale` | :heavy_check_mark: | `number` | `UINT8` |
 
-`Scale` is the price's scaling factor.
+- `TransactionType` Indicates a new transaction type `OracleSet`. The integer value is TBD.
+- `Account` is the XRPL account that has update and delete privileges on the Oracle being set.
```

Comment:
We should mention (here and elsewhere, in the other txn sections) that this field corresponds to the `Owner` field on the `PriceOracle` ledger object.